MIGRATION.md

```diff
@@ -0,0 +1,89 @@
+# Migrating to the `cloudant-node-sdk` library
+This document is to assist in migrating from the `nodejs-cloudant` (module: `cloudant`) to the newly supported [`cloudant-node-sdk`](https://github.com/IBM/cloudant-node-sdk) (package: `@ibm-cloud/cloudant`). Compatible with Javascript and TypeScript.
+
+## Initializing the client connection
+There are several ways to create a client connection in `cloudant-node-sdk`:
+1. [Environment variables](https://github.com/IBM/cloudant-node-sdk#authentication-with-environment-variables)
+2. [External configuration file](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration)
+3. [Programmatically](https://github.com/IBM/cloudant-node-sdk#programmatic-authentication)
+
+[See the README](https://github.com/IBM/cloudant-node-sdk#code-examples) for code examples on using environment variables.
+
+## Other differences
+1. For storing credentials in a file, `dotenv` package not needed anymore. Check the new operation [here](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration).
```

Comment:
How about:
```suggestion
1. For storing credentials in a file, the `dotenv` package is not supported. See the [external file configuration section](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration) in our API docs for handling this feature in our new library.
```

Reply:
For me `the `dotenv` package is not supported.` says that we deal with issues related to `dotenv` but this is not the case. This is just a suggested package in this situations. So I am reluctant about the first part. 

WDYT about this:
```suggestion
1. For storing credentials in a file has changed, the `dotenv` package is not needed anymore. See the [external file configuration section](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration) in our API docs for the new handling.
```

Reply:
Changed in c706430